x-pack/plugins/cloud_security_posture/server/lib/check_index_status.ts

```diff
@@ -13,21 +13,42 @@ export const checkIndexStatus = async (
   esClient: ElasticsearchClient,
   index: string,
   logger: Logger,
-  postureType?: PostureTypes
+  postureType?: PostureTypes,
+  retentionTime?: number
```

Comment:
let's maybe turn the last two params into `options` param? then to function calls would be much more clear, eg
instead of `checkIndexStatus(esClient, LATEST_FINDINGS_INDEX_DEFAULT_NS, logger, 'all', 72)` (it's really hard to understand what this 72 means)
it will be 
```
checkIndexStatus(esClient, LATEST_FINDINGS_INDEX_DEFAULT_NS, logger, {
  postureType: 'all', 
  retentionPeriod: 72
})
```

I'd also change `retentionPeriod` to a string and would pass it as `72h` so it's much more clear what is the time dimension it is, like that
```
checkIndexStatus(esClient, LATEST_FINDINGS_INDEX_DEFAULT_NS, logger, {
  postureType: 'all', 
  retentionPeriod: '72h'
})
```